subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,76 @@
+
+#### Comprehension
+
+##### Describe the microservices architecture implemented.
+
+###### Did the architecture effectively split functionalities into different microservices based on key factors like scalability and domain separation?
+
+##### Ask the student to explain one of the Ansible playbook
+
+###### did he/she clearly explain all the Ansible playbook
+
+##### Discuss the CI/CD pipeline setup.
+
+###### Was the CI/CD pipeline, integrating Jenkins and SonarQube, effectively used to improve the development process?
+
+##### Detail the security measures implemented.
+
+###### Were comprehensive security measures like SSL/TLS, secret management, and principle of least privilege correctly implemented?
+
+##### Explain the database schema for PostgreSQL and Neo4j.
+
+###### Did the data structure in PostgreSQL and Neo4j effectively support the application's requirements?
+
+#### Functional
+
+##### Verify the execution of Ansible playbooks.
+
+###### Did the Ansible playbooks execute without errors and configure the environment as intended?
+
+###### Were the playbooks able to handle re-running scenarios without causing disruptions or inconsistencies?
+
+##### Test each microservice API.
```

Comment:
I would add more information here: what are the minimum APIs expected or what are the minimum type of interactions expected? For example:
- there must be an endpoint to add a travel and it must be possible to use it only with the proper authentication system enabled
- there must be an endpoint to retrieve all the travel for the logged user 
- there must be an endpoint to retrieve all the travel for all the users, only accessible for admins 
- (all the other minimum endpoints required from the application)